lib/bulbs-elements/util/resize-parent-frame.js

```diff
@@ -1,19 +1,34 @@
 // Resize a (Kinja) parent frame, using Kinja's parent notification support.
 
-export default function resizeParentFrame (height) {
-  if (window.parent) {
-    // We're in a child frame (e.g. an embed)
+function sendResizeMessage () {
+
+  // Remove any anchor tags
+  let sourceUrl = window.location.href.split('#')[0];
 
-    // Remove any anchor tags
-    let sourceUrl = window.location.href.split('#')[0];
+  let height = document.body.clientHeight;
 
-    window.parent.postMessage({
-      kinja: {
-        sourceUrl,
-        resizeFrame: {
-          height,
-        },
+  console.log('Resize parent frame', height);
```

Comment:
Should remove this `console.log` prior to merging.

Reply:
Good catch